mono/profiler/mono-profiler-log.h

```diff
@@ -153,4 +156,108 @@ enum {
 	SAMPLE_LAST
 };
 
+
+// If you alter MAX_FRAMES, you may need to alter SAMPLE_BLOCK_SIZE too.
+#define MAX_FRAMES 32
+
+typedef enum {
+	DomainEvents = 1 << 0,
```

Comment:
Is there a reason for this enum naming style? It deviates from the style we use throughout most of the runtime AFAIK. I was actually a bit confused what `DomainEvents` was when I was reading the arg-parsing.c code above. My first thought was "what is a type name doing here?".